src/main/java/edu/lwtech/csd297/tap4tap/pojos/Brewery.java

```diff
@@ -13,13 +13,13 @@ public class Brewery {
     private String country;
     private String websiteUrl;
     private String phone;
-    private String longitude;
-    private String latitude;
+    private float longitude;
+    private float latitude;
```

Comment:
We should probably make this double instead of float because the database storing the latitude and longitude as fixed-point values. Double will still result in some loss of precision but will be more precise than float.

Reply:
Done.